src/components/VPopover/VPopoverContent.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <div
+    v-if="visible"
+    ref="popoverRef"
+    :class="$style.popover"
+    :aria-hidden="!visible"
+    :tabindex="typeof $props.tabindex !== 'undefined' ? $props.tabindex : -1"
+    v-on="$listeners"
+    @keydown="onKeyDown"
+    @blur="onBlur"
+  >
+    <slot />
+  </div>
+</template>
+
+<script>
+import { defineComponent, toRefs, ref } from '@nuxtjs/composition-api'
+import { useFocusOnShow } from '~/composables/use-focus-on-show'
+import { useFocusOnHide } from '~/composables/use-focus-on-hide'
+import { useHideOnClickOutside } from '~/composables/use-hide-on-click-outside'
+import { useFocusOnBlur } from '~/composables/use-focus-on-blur'
+import { usePopper } from '~/composables/use-popper'
+import { warn } from '~/utils/warn'
+
+import { propTypes } from './VPopoverContent.types'
+
+export default defineComponent({
+  name: 'VPopover',
+  props: propTypes,
+  /**
+   * This is the only documented emitted event but in reality we pass through `$listeners`
+   * to the underlying element so anything and everything is emitted. `@keydown` is the
+   * only one this component overrides and controls (but ultimately still emits).
+   */
+  emits: ['keydown', 'blur'],
+  /**
+   * @param {import('./VPopoverContent.types').Props} props
+   * @param {import('@nuxtjs/composition-api').SetupContext} context
+   */
+  setup(props, { emit, attrs }) {
+    if (!attrs['aria-label'] && !attrs['aria-labelledby']) {
+      warn('You should provide either `aria-label` or `aria-labelledby` props.')
+    }
+
+    const propsRefs = toRefs(props)
+    const popoverRef = ref()
+    const focusOnBlur = useFocusOnBlur({
+      popoverRef,
+      popoverPropsRefs: propsRefs,
+    })
+
+    useFocusOnShow({
+      popoverRef,
+      popoverPropsRefs: propsRefs,
+    })
+    useFocusOnHide({
+      popoverRef,
+      popoverPropsRefs: propsRefs,
+    })
+    useHideOnClickOutside({
+      popoverRef,
+      popoverPropsRefs: propsRefs,
+    })
+
+    usePopper({
+      popoverRef,
+      popoverPropsRefs: propsRefs,
+    })
+
+    /**
+     * @param {Event} event
```

Comment:
Hmm, it shouldn't reference either, it should reference the global `Event` type from TypeScript's built in `dom.d.ts`: https://github.com/microsoft/TypeScript/blob/main/lib/lib.dom.d.ts#L5034

When I hover over it it shows the description from the global dom types but if I "go to definition" on it in VSCode it does show React and the global dom types. I think the React types get detected because of interface merging and because React declares global types. Luckily they don't modify the Event type in this way. I have no idea what WebStorm pulls types from sentry without an explicit reference to it.